src/p2p/net_node.inl

```diff
@@ -799,14 +799,16 @@ namespace nodetool
           "4pixvbejrvihnkxmduo2agsnmc3rrulrqc7s3cbwwrep6h6hrzsibeqd.onion:18083"
         };
       }
+      break;
```

Comment:
These should be `return {};` to indicate a valid zone value but no available seed nodes.